srcpkgs/python3-aiostream/template

```diff
@@ -0,0 +1,13 @@
+# Template file for 'python3-aiostream'
+pkgname=python3-aiostream
+version=0.4.5
+revision=1
+build_style=python3-module
+hostmakedepends="python3-setuptools"
+depends="python3"
+short_desc="Generator-based operators for asynchronous iteration"
+maintainer="joelchrono12 <joel.chrono@disroot.org>"
+license="GPL-3.0-only"
+homepage="https://aiostream.readthedocs.io/"
+distfiles="${PYPI_SITE}/a/aiostream/aiostream-${version}.tar.gz"
```

Comment:
Only the github tarball has tests
```suggestion
distfiles="distfiles="https://github.com/vxgmichel/aiostream/archive/refs/tags/v${version}.tar.gz"
checkdepends="python3-pytest python3-pytest-asyncio"
```
this will fail with --cov not being available in setup.cfg, but we don't really need coverage reports, so I would just disable them with patch/vsed.